internal/services/compute/linux_virtual_machine_resource_other_test.go

```diff
@@ -2638,3 +2711,149 @@ resource "azurerm_linux_virtual_machine" "test" {
 }
 `, r.template(data), data.RandomInteger)
 }
+
+func (r LinuxVirtualMachineResource) otherBypassPlatformSafetyChecksOnUserSchedule(data acceptance.TestData, enabled bool) string {
```

Comment:
By the looks of it this property requires a maintenance schedule attached to the VM i.e. `azurerm_maintenance_assignment_virtual_machine` and `azurerm_maintenance_configuration` with an explicit `depends_on`. Can you update to test configs to include those to validate that this feature works with the right conditions?

Reply:
@stephybun the maintenance schedule is not a hard requirement, there are user scenarios where the schedule is not attached: https://learn.microsoft.com/azure/update-center/prerequsite-for-schedule-patching?tabs=new-prereq-rest-api%2Cauto-portal#user-scenarios